app.js

```diff
@@ -0,0 +1,26 @@
+const express = require("express");
+const { getCategories } = require("./controllers/categories.controller");
+const { getReviews } = require("./controllers/reviews.controller");
+const app = express();
+
+app.get("/api/categories", getCategories);
+app.get("/api/reviews/:review_id",getReviews)
+
+app.use((err, req, res, next) => {
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  } else {
+    next(err);
+  }
+});
+
+app.use((err, req, res, next) => {
```

Comment:
The 500 error should always be the last in this chain as it's the last conclusion an error is likely to come to

Reply:
Thank you !